src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectDebuggerProvider.cs

```diff
@@ -68,16 +68,17 @@ public override Task<bool> CanLaunchAsync(DebugLaunchOptions launchOptions)
         /// <summary>
         /// Helper returns the correct debugger engine based on the targeted framework
         /// </summary>
-        public static Guid GetManagedDebugEngineForFramework(string targetFramework)
+        public static Guid GetManagedDebugEngineForFramework(string targetFramework, bool mixedMode)
         {
             // The engine depends on the framework
             if (IsDotNetCoreFramework(targetFramework))
             {
+                if (mixedMode) throw new Exception(VSResources.UnmanagedDebuggingNetCoreNotSupported);
```

Comment:
@srivatsn looking at the `DebuggingEngines` type, I see this:
```
              public static readonly Guid ManagedLegacyEngine;
        public static readonly Guid SqlEngine;
        public static readonly Guid NativeOnlyEngine;
        public static readonly Guid ManagedOnlyEngine;
        public static readonly Guid MixedNativeAndManagedEngine;
        public static readonly Guid ScriptEngine;
        public static readonly Guid GPUDebugEngine;
        public static readonly Guid ManagedCoreEngine;

```

I didn't see one for `MixedNativeAndCoreEngine`?